app/controllers/queue_items_controller.rb

```diff
@@ -17,11 +17,46 @@ def create
 
     redirect_to user_queue_items_path(current_user)
   end
+require 'pry'
+  def update
+    begin
+      QueueItem.transaction do
+        all_updates = params[:queue_items]
+        all_updates.each do |update|
+          queue_item = QueueItem.find(update[:id])
+          if queue_item.user == current_user
+            review = Review.find_by_video_id_and_user_id(queue_item.video.id,queue_item.user.id)
```

Comment:
using dynamic `find_by...` is not advised and not recommended. You should use standard `find_by` which is a method in ActiveRecord. The former gets dynamically built and uses method missing - makes it inefficient.